packages/venia-concept/src/components/MiniCart/__tests__/product.spec.js

```diff
@@ -61,33 +71,36 @@ test('passed functions are called from nested `Section` components', () => {
     expect(removeItem).toHaveBeenCalled();
 });
 
-test('Product variants are rendered', () => {
+test('Product name is rendered', () => {
     const wrapper = shallow(
-        <Product item={item} currencyCode={'EUR'} totalsItems={totalsItems} />
+        <Product
+            item={item}
+            currencyCode={'EUR'}
+            totalsItems={totalsItems}
+            classes={classes}
+        />
     ).dive();
 
-    // The product's name is in the div at index 1.
     expect(
         wrapper
-            .find('div')
-            .at(1)
+            .find(`.${classes.name}`)
+            .at(0)
             .text()
-    ).toEqual(item.name);
+    ).toContain(item.name);
+});
+
+test('Product variants are rendered', () => {
+    const wrapper = shallow(
+        <Product
+            item={item}
+            currencyCode={'EUR'}
+            totalsItems={totalsItems}
+            classes={classes}
+        />
+    ).dive();
 
-    // The product variants are rendered.
-    const expectedOptions = JSON.parse(totalsItems[0].options);
-    expectedOptions.forEach((option, index) => {
-        expect(
-            wrapper
-                .find('dt')
-                .at(index)
-                .text()
-        ).toContain(option.label);
-        expect(
-            wrapper
-                .find('dd')
-                .at(index)
-                .text()
-        ).toEqual(option.value);
+    wrapper.find(`.${classes.optionLabel}`).forEach((l, i) => {
```

Comment:
I'm fine with `i`, but let's change `l` to something more meaningful. I'm assuming it means `label`?